lib/msal-browser/src/utils/PopupUtils.ts

```diff
@@ -9,6 +9,16 @@ import { BrowserAuthError } from "../error/BrowserAuthError";
 import { AuthorizationUrlRequest } from "../request/AuthorizationUrlRequest";
 import { BrowserConstants, InteractionType, TemporaryCacheKeys } from "./BrowserConstants";
 
+/**
+ * Popup configurations for setting dimensions and position of popup window
+ */
+export type PopupConfiguration = {
+    height: number;
+    width: number;
+    top: number;
+    left: number;
```

Comment:
One question I have is whether all of these should be required, or if we can make some of them optional. For example, if an app provides just height and width, we can assume they want it centered, and use the logic we have today to center it for them.

Reply:
(I see Peter made a similar suggestion below)

Reply:
What if we make this 2 types `PopupSize` (height, width) and `PopupPosition` (top, left) where both are optional but the values inside each are required. I don't think someone would provide just the height but not the width, for example, right?

Reply:
I like that, I changed it to `PopupWindowAttribute` with the optional `PopupSize` and `PopupPosition`, with values required.